<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PressKeyCmdWidget</class>
 <widget class="QWidget" name="PressKeyCmdWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>262</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>10</width>
    <height>10</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>4000</width>
    <height>4000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PressKeyCmdWidget</string>
  </property>
  <widget class="QKeySequenceEdit" name="keySequenceEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>5</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enter a key</string>
   </property>
  </widget>
  <widget class="QLabel" name="pressKeyLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>4</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Press</string>
   </property>
  </widget>
  <widget class="QComboBox" name="_specialKeyComboBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>89</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="maxVisibleItems">
    <number>20</number>
   </property>
   <property name="maxCount">
    <number>100</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Alt+Tab</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Alt+F4</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="_specialKeyCheckBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>89</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Special keys</string>
   </property>
  </widget>
  <widget class="QPushButton" name="keySequenceClearButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>6</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QListWidget" name="modifierListWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>5</y>
     <width>135</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::CopyAction</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
   <property name="movement">
    <enum>QListView::Static</enum>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
   <property name="uniformItemSizes">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="selectionRectVisible">
    <bool>true</bool>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
   <item>
    <property name="text">
     <string>Ctrl</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Shift</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Alt</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="letterLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>3</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="toolTip">
    <string>The key to press</string>
   </property>
   <property name="maxLength">
    <number>1</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="plusLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>5</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="keyTypeComboBox">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>3</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>letter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>key</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>special key</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="oldKeyBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="maxVisibleItems">
    <number>20</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Escape</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Backspace</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tab</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Return</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pause</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Caps Lock</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Space</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Page Up</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Page Down</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>End</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Home</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Insert</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Delete</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Arrow Up</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Arrow Down</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Arrow Left</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Arrow Right</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NUM Lock</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Scroll Lock</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Window</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="specialKeyComboBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>3</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="maxVisibleItems">
    <number>20</number>
   </property>
   <property name="maxCount">
    <number>100</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tab</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Print</string>
    </property>
   </item>
  </widget>
  <zorder>specialKeyComboBox</zorder>
  <zorder>oldKeyBox</zorder>
  <zorder>pressKeyLabel</zorder>
  <zorder>_specialKeyComboBox</zorder>
  <zorder>_specialKeyCheckBox</zorder>
  <zorder>modifierListWidget</zorder>
  <zorder>letterLineEdit</zorder>
  <zorder>plusLabel</zorder>
  <zorder>keyTypeComboBox</zorder>
  <zorder>keySequenceEdit</zorder>
  <zorder>keySequenceClearButton</zorder>
 </widget>
 <tabstops>
  <tabstop>_specialKeyComboBox</tabstop>
  <tabstop>_specialKeyCheckBox</tabstop>
  <tabstop>keySequenceEdit</tabstop>
  <tabstop>keySequenceClearButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
